packages/iTwinUI-react/src/core/Footer/Footer.tsx

```diff
@@ -28,6 +30,11 @@ export type FooterProps = {
    * Provide localized strings.
    */
   translatedTitles?: TitleTranslations;
+  /**
+   * Custom footer content. If provided, it will render only what you passed.
+   * If function provided, it should accept an array of default footer elements and return `React.ReactNode`.
+   */
+  children?: React.ReactNode | ((elements: FooterElement[]) => React.ReactNode);
```

Comment:
What does this render function provide that is not possible with just regular children?

Reply:
It provides default footer elements else if you want to customize anything a bit, you would need to take them from somewhere.

Reply:
What is the benefit of doing that?

Reply:
if we change something about the default elements in the future, we won't need to think about `children` handling

Reply:
We won't be able to make this change without major change anyways.

Reply:
Because I don't like this not working code either 
![image](https://user-images.githubusercontent.com/36186912/182179022-d40a0538-decc-4440-8c8a-ef7cd2db4d01.png)

Reply:
Not sure I understand that code snippet. Where is that from?

Reply:
Changes required to reuse default elements if we want to export them.

Reply:
Did something